spec/factories/notifications.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :notification do
+    sender { association(:user) }
```

Comment:
association :sender, factory: :user
association :recipient, factory: :user

Reply:
@XaoGao both gives the same result, am I right?